app/models/namespace.rb

```diff
@@ -120,6 +120,20 @@ def self.make_valid(name)
     name = final[0..MAX_NAME_LENGTH]
 
     return nil if name !~ NAME_REGEXP
+
+    # To avoid any name conflict we append an incremental number to the end
+    # of the name returns it as the name that will be used on both Namespace
+    # and Team on the User#create_personal_namespace! method
+    # TODO: workaround until we implement the namespace/team removal
+    if name =~ NAME_REGEXP
```

Comment:
Since you have a `return nil if name !~ NAME_REGEXP` as the previous statement, I think it's safe to skip this check.